packages/amplication-git-utils/src/utils/prepare-files-for-pull-request.ts

```diff
@@ -29,34 +29,36 @@ export async function prepareFilesForPullRequest(
 
   const authFolder = "server/src/auth/";
 
-  const files: Required<Changes["files"]> = Object.fromEntries(
-    pullRequestModule.map((module) => {
-      // ignored file
-      if (amplicationIgnoreManger.isIgnored(module.path)) {
-        return [join(AMPLICATION_IGNORED_FOLDER, module.path), module.code];
-      }
-      // Deleted file
-      if (module.code === null) {
-        return [module.path, module.code];
-      }
-      // Regex ignored file
-      if (
-        !module.path.startsWith(authFolder) &&
-        doNotOverride.some((rx) => rx.test(module.path))
-      ) {
-        return [
-          module.path,
-          ({ exists }) => {
-            // do not create the file if it already exist
-            if (exists) return null;
+  const files: File[] = pullRequestModule.map((module) => {
+    // ignored file
+    if (amplicationIgnoreManger.isIgnored(module.path)) {
+      return [
+        {
+          file: join(AMPLICATION_IGNORED_FOLDER, module.path),
+          content: module.code,
+        },
+      ];
```

Comment:
why are we returning an array? I would expect the following:
```suggestion
      return {
          file: join(AMPLICATION_IGNORED_FOLDER, module.path),
          content: module.code,
        };
```